hassio/src/system/hassio-supervisor-info.ts

```diff
@@ -233,6 +272,21 @@ class HassioSupervisorInfo extends LitElement {
     button.progress = false;
   }
 
+  private async _supervisorRestart(ev: CustomEvent): Promise<void> {
+    const button = ev.currentTarget as any;
+    button.progress = true;
+
+    try {
+      await restartSupervisor(this.hass);
+    } catch (err) {
+      showAlertDialog(this, {
+        title: "Failed to restart the supervisor",
+        text: extractApiErrorMessage(err),
+      });
+    }
+    button.progress = false;
```

Comment:
Move this to finally

Reply:
is finally executed regardless of the outcome?